### PR TITLE
translate(pt/developer-docs/hit-counts)

### PR DESCRIPTION
**Pull request by G3mha**
Translates the file `docs/pt/developer-docs/hit-counts.md` to pt-BR, and adds it to the sidebar.

### CODE REVIEW

File: docs/pt/developer-docs/hit-counts.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: Contagem de Acertos
+description: Aprenda a usar Contagem de Acertos no Editor de Conquistas para criar condições que precisam ser cumpridas múltiplas vezes antes de engatilhar uma conquista.
+---
+
+# Contagem de Acertos
+
+No Editor de Conquistas, o campo no lado mais à direita é a Contagem de Acertos.
+
+![image](https://user-images.githubusercontent.com/32706333/48969526-38774c80-efbd-11e8-98ec-9d6c16dba022.png)
```
Comment (wescopeland) [inline docs/pt/developer-docs/hit-counts.md]: I know this is a bit out of scope, but this image needs to be fixed.

While we're in here, would you be willing to add this image to the repo itself (similar to other images in the public directory)?

Comment (G3mha) [reply docs/pt/developer-docs/hit-counts.md]: Certainly! Should I do that for all the other images like this one that I bump into?

**Comment by G3mha** [comment]
@wescopeland I noticed that there are many images hosted on GitHub's user content CDN. Would you like me to help migrate other similar images to the public directory as I find them?

And btw, this PR is ready for re-review.

**Comment by wescopeland** [comment]
> Would you like me to help migrate other similar images to the public directory as I find them?

Yes, that would be quite beneficial. Thank you!

**Review by wescopeland — APPROVED** [review]

**Pull request merged**